client/components/cart-songs-view.js

```diff
@@ -0,0 +1,94 @@
+import React, {Component} from 'react'
+import {connect} from 'react-redux'
+import {fetchCart, setLocalStorageItem} from '../store/cart'
+import {
+  Button,
+  Card,
+  Container,
+  Media,
+  Row,
+  Col,
+  Image,
+  ListGroup
+} from 'react-bootstrap'
+
+class Cart_Songs_View extends React.Component {
+  constructor() {
+    super()
+  }
+
+  async componentDidMount() {
+    await this.props.getCart()
+    await this.props.setLocalStorage(this.props.currentCart)
+  }
+
+  render() {
+    const cart = this.props.currentCart
+    const cartSongs = cart.songs
+
+
+    return (
+      <Container fluid>
+        <div>
+          <section>
+            <h3>Songs:</h3>
+            <ListGroup className="list-unstyled">
+              {cartSongs ? (
+                <div>
+                  {cartSongs.map(song => (
```

Comment:
Don't think we need this container, since  the parent component already has the structure we need and the map function is also in there, so all we need for this component to be is just a list item

Reply:
Removed everything before/after the ListGroup Component